src/contracts/chainlinkOracle.ts

```diff
@@ -8,5 +8,12 @@ export const chainlinkOracle = {
     address: '0x508d4AC10F057beACb7Cef6f112e79075045C3C9',
     name: 'Decentraland ChainLink Oracle',
     chainId: ChainId.MATIC_MUMBAI
+  },
+  [ChainId.MATIC_MAINNET]: {
+    version: '1',
+    abi: abis.ChainlinkOracle,
+    address: '0xe18B1361d41afC44658216F3Dc27e48c2336e3c2',
+    name: 'Decentraland ChainLink Oracle',
```

Comment:
It might be a good idea to abstract the name of the contracts into a variable as it is being used repeatedly.

Reply:
Good idea